src/nyx/phase1.cpp

```diff
@@ -44,8 +44,9 @@ namespace Nyxus
 			lyr = 0; //	Layer
 
 		// Open an image pair
-		ImageLoader imlo;
-		bool ok = imlo.open (intens_fpath, label_fpath);
+		ImageLoader & imlo = theImLoader;	//???	ImageLoader imlo;
+		bool ok = true;	//???	bool ok = imlo.open (intens_fpath, label_fpath);

```

Comment:
don't we need to call imlo.open(...) ?

Reply:
The loader is being open and closed earlier, in scan_fastloader_way.cpp/processDataset(), so there's no need for calling open() here on the same global image loader instance. I cleaned the code in order for it to look less confusing